src/main/resources/data.sql

```diff
@@ -12,15 +60,72 @@ CREATE TABLE Customers (
     FOREIGN KEY (cid) REFERENCES Users (uid) ON DELETE CASCADE
 );
 
+CREATE TABLE FDSManagers (
+	fmid INTEGER PRIMARY KEY,
+	FOREIGN KEY (fmid) REFERENCES Users (uid) ON DELETE CASCADE
+);
+
+CREATE TABLE Promotions (
+	pid INTEGER PRIMARY KEY,
+	startDate TIMESTAMP,
+	endDate TIMESTAMP,
+	promotionType VARCHAR(100),
+	discount NUMERIC
+);
+
+-- pid = id of promotion offered by restaurant
+CREATE TABLE Restaurants (
+	rid INTEGER PRIMARY KEY,
+	name VARCHAR(100),
+	mininumPurchase MONEY,
+	pid INTEGER,
+	FOREIGN KEY (pid) REFERENCES promotions
+);
+
+CREATE TABLE RestaurantStaff (
+	rsid INTEGER PRIMARY KEY,
+	employedBy INTEGER NOT NULL,
+	FOREIGN KEY (rsid) REFERENCES Users (uid) ON DELETE CASCADE,
+	FOREIGN KEY (employedBy) REFERENCES Restaurants (rid)
+);
+
+CREATE TABLE FoodItems (
+	fid INTEGER PRIMARY KEY,
+	rid INTEGER,
+	name VARCHAR(100),
+	category VARCHAR(100),
+	price MONEY,
+	dailyLimit INTEGER,
+	FOREIGN KEY (rid) REFERENCES Restaurants ON DELETE CASCADE
+);
+
+CREATE TABLE Reviews (
+	rvid INTEGER PRIMARY KEY,
+	rating INTEGER,
+	content VARCHAR(1000)
+);
+
 CREATE TABLE Orders (
     oid INTEGER PRIMARY KEY,
+	drid INTEGER,
+	rvid INTEGER,
     totalCost INTEGER,
-    deliveryFee INTEGER,
+    deliveryFee MONEY,
     paymentType INTEGER,
     location VARCHAR(100),
     orderTime TIMESTAMP,
     departureTime TIMESTAMP,
     restaurantArrivalTime TIMESTAMP,
     restaurantDepartureTime TIMESTAMP,
-    deliveryTime TIMESTAMP
-)
+    deliveryTime TIMESTAMP,
+	FOREIGN KEY (drid) REFERENCES DeliveryRiders,
+	FOREIGN KEY (rvid) REFERENCES Reviews
+);
+
+CREATE TABLE OrdersContains (
```

Comment:
I think we can do away with `OrderContains` and use the `Reviews` table for this instead